Applications/ReinforcementLearning/DeepQ/jni/main.cpp

```diff
@@ -339,7 +339,8 @@ int main(int argc, char **argv) {
         nntrainer::Tensor in_tensor;
         nntrainer::sharedConstTensor test;
         try {
-          in_tensor = nntrainer::Tensor({input});
+          in_tensor = nntrainer::Tensor(
```

Comment:
Tensor Object creation code is modified to add the default format & type ({nntrainer::Tformat::NCHW, nntrainer::Tdatatype::FP32}) in many places in this PR.

I believe, as we have handled the default values in constructor, do we need explict change here & everywhere?

Reply:
yes. we will check it in late pr. Thanks.